### PR TITLE
SEO-110 Update selenium test for HTML title

### PR DESCRIPTION
**Pull request by gabrys**

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java [hunk 1]
```diff
@@ -14,240 +14,227 @@
  */
 public class HtmlTitleTests extends NewTestTemplate {
 
+  // TODO: QAART-681 Sonar should not report duplicate strings warnings for data providers
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java [hunk 2]
```diff
@@ -14,240 +14,227 @@
...
+            "Ulica Kazimierza Wielkiego – Poznańska Wiki – Wikia",
+        },
+        // Custom title
         {
             TEST_WIKI_CUSTOM_TITLE,
             "WikiDex",
             "WikiDex, la enciclopedia Pokémon - Wikia",
-            "WikiDex - WikiDex - Wikia"
         },
+        // TODO: SEO-77 Fix the title of the main pages for Mercury
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java [hunk 3]
```diff
@@ -14,240 +14,227 @@
...
 
   Credentials credentials = Configuration.getCredentials();
 
   @DataProvider
-  private Object[][] dataHtmlTitleTest001() {
+  private Object[][] dataHtmlTitleTest() {
     return new Object[][]{
-        // original title
+        // Original title
+        // TODO: SEO-77 Fix the title of the main pages for Mercury
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java]: INFO Complete the task associated to this TODO comment. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* INFO 3 info

Watch the comments in this conversation to review them.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* INFO 3 info

Watch the comments in this conversation to review them.

**Pull request merged**